src/components/specific/files/file-upload-card/FileUploadCard.vue

```diff
@@ -101,63 +101,64 @@ export default {
     let lastProgressTime = null;
     const progress = reactive({
       percentage: 0,
-      uploaded: 0,
-      fileUploaded: {},
+      fileUploaded: 0,
       folderUploaded: 0,
-      onGoingUploadsCounter: 0,
+      onGoingUploads: {},
```

Comment:
I'm curious if using a Map here (https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Map) would lead to a more readable code.